<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <title>SHAPEwarp</title>
        <link rel="shortcut icon" href="https://www.incarnatolab.com/favicon.ico" type="image/x-icon">
        <link rel="stylesheet" href="css/styles.css">
        <link href="https://fonts.googleapis.com/css?family=Merriweather:400,400i|Orbitron:900|Raleway:300,400,400i,600,700" rel="stylesheet">
        <script src="https://code.jquery.com/jquery-3.6.4.min.js"></script>
    </head>
    <body>
        <div id="menu-bar" class="menu-bar">
            <div class="container">
                <nav>
                    <ul>
                        <li><a href="index.html">Submit a job</a></li>
                        <li><a href="https://github.com/dincarnato/SHAPEwarp/">Standalone</a></li>
                        <li><a href="help.html">Help</a></li>
                    </ul>
                </nav>
            </div>
        </div>
        <header>
            <div class="container">
                <h1 class="page-title"><span class="page-title2">SHAPE</span>WARP</h1>
            </div>
        </header>
        <section id="main-content">
            <div class="container">
                <h1 class="content-title">What is SHAPEwarp?</h1>
                <p>
                    <strong>SHAPEwarp</strong> is a sequence-agnostic method for the identification of structurally-similar 
                    RNA elements in a database of SHAPE (Selective 2&prime;-hydroxyl acylation analyzed by primer extension) chemical 
                    probing-derived reactivity profiles.
                    The approach used by SHAPEwarp is inspired by the <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a> 
                    algorithm and builds on top of two widely used methods for similarity search in time series data: Mueen's 
                    Algorithm for Similarity Search (<a href="https://www.cs.unm.edu/~mueen/FastestSimilaritySearch.html">MASS</a>) 
                    and dynamic time warping (<a href="https://en.wikipedia.org/wiki/Dynamic_time_warping">DTW</a>).<br/><br/>
                    Briefly, SHAPEwarp identifies ungapped groups of short segments (<i>High Scoring Groups</i>, or <i>HSGs</i>) 
                    showing highly similar reactivity profiles between a query and a target RNA. Each HSG serves as a seed from 
                    which the alignment is bidirectionally extended using a banded semi-global alignment algorithm that incorporates 
                    both features of the Gotohâ€™s Smith-Waterman algorithm with affine gap penalties and of DTW.<br/>
                    By default, the alignment solely relies on the reactivity profiles, although sequence can be optionally considered 
                    (see below <a href="#search-parameters"><strong>Search parameters</strong></a> and 
                    <a href="#understanding-search-parameters"><strong>Understanding search parameters</strong></a>). 
                </p>
                <p>
                    For a complete description of the algorithm, please refer to the original publication:<br/><br/>
                    <a style="text-decoration: none;" href="https://www.nature.com/articles/s41467-022-29398-y">Edoardo Morandi, Martijn J. van Hemert &amp; Danny Incarnato (2022) <strong>SHAPE-guided RNA structure homology search and motif discovery</strong> (Nat. Commun.)</a>
                </p>
                <br/>
                <h1 class="content-title">How does SHAPEwarp work?</h1>
                <p>
                    SHAPEwarp takes a user-provided query, consisting of an RNA sequence and a corresponding set of SHAPE reactivities,
                    and searches it against a database of SHAPE reactivity profiles.<br/>
                    A query is composed of 3 lines, respectively containing:
                </p>
                <p>
                    <ol>
                        <li>Unique query ID</li> 
                        <li>Sequence</li> 
                        <li>Comma-separated list of SHAPE reactivities</li> 
                    </ol>
                </p>
                <p>Below you can see an example:</p>
                <p>
                    <pre>Ecoli_16S
GATGTGAAATCCCCGGGCTCAACCTGGGAACTGCATCTGATACTGGCAAGCTTGAGTCTCGTAGAGGGGGGTAGAATTCCAGGTGTAGCGGTGAAATGCGTAGAGATCTGGAGGAATACCGGTGGCGAAGGCGGCCCCCTGGACGAAGACTGACGCTCAGGTGCGAAAGCGTGGGGAGCAAACAGGATTAGATACCCTGG
0.052,0.046,0.108,0.241,0.221,1.224,0.246,0.846,1.505,0.627,0.078,0.002,0.056,0.317,0.114,0.157,0.264,1.016,2.925,2.205,1.075,1.210,0.191,0.016,0.045,0.015,0.087,0.572,0.052,0.157,0.796,2.724,0.027,0.000,0.000,0.000,0.000,0.000,0.004,0.003,0.063,0.144,0.072,0.054,0.096,0.112,0.002,0.000,0.019,0.026,0.021,1.022,2.108,0.111,0.000,0.007,0.000,0.002,0.000,0.010,0.037,0.078,0.152,0.355,1.738,0.715,0.211,0.179,0.036,0.046,0.159,0.257,0.312,0.931,0.798,0.618,0.935,0.147,0.015,0.014,0.031,0.147,0.149,0.577,1.052,1.410,0.487,0.636,0.238,0.286,0.462,1.586,1.683,0.597,1.165,1.265,2.094,0.422,0.462,1.900,4.055,0.481,0.511,0.087,1.217,1.180,0.094,0.018,0.033,0.081,0.148,0.163,0.160,1.019,0.339,0.507,1.039,0.824,0.122,0.420,0.429,0.913,1.383,0.610,0.417,0.825,0.743,0.433,0.401,0.993,0.497,0.404,0.407,0.316,0.017,0.005,0.046,0.072,0.037,0.091,0.282,0.203,0.033,0.004,0.021,0.262,0.157,0.050,0.019,0.059,0.102,0.083,0.066,0.040,0.075,0.061,0.573,0.631,0.427,0.265,1.190,0.066,0.042,0.085,0.424,0.413,0.375,0.447,0.035,0.045,0.037,0.242,0.221,0.157,0.170,0.370,1.238,0.743,0.571,0.138,0.837,0.859,0.042,0.021,0.080,0.318,0.195,0.792,1.581,1.058,2.004,1.512,2.273,1.256,0.036,0.005,0.094,0.091,0.464,0.741</pre>
                </p>
                <p>
                    Multiple queries can be simultaneously searched against a database.<br/>
                    The webserver has the following limits:
                </p>
                <p>
                    <ul>
                        <li>Each individual query cannot exceed <strong>300 nucelotides</strong></li> 
                        <li>The sum of all queries cannot exceed <strong>3,000 nucleotides</strong></li> 
                    </ul>
                </p>
                <p>
                    To perform a test run, just click on "<strong>Submit a job</strong>" in the top menu, then click on 
                    "<strong>Load sample query</strong>", then click "<strong>Submit</strong>" (see below):
                    <br/><br/>
                    <img src="images/test_run.png" alt="Test run"/>
                </p>
                <br/>
                <h1 class="content-title" id="search-parameters">Search parameters</h1>
                <p>
                    Below is a complete list and description of SHAPEwarp's parameters. To better understand how each parameter is used,
                    check the next section <a href="#understanding-search-parameters"><strong>Understanding search parameters</strong></a>.
                </p>
                <br/>
                <p>
                    <ul>
                        <strong><u>Basic options</u></strong>
                        <br/><br/>
                        <ul>
                            <li><strong>Inclusion E-value</strong>. Matches having E-value &lt; this threshold are considered to be significant</li>
                            <li>
                                <strong>Report E-value</strong>. Matches having E-value &ge; <strong>Inclusion E-value</strong> and &lt; this threshold are reported,
                                but no details (such as sequence/reactivity alignments, nor consensus structure) are reported
                            </li>
                            <li>
                                <strong>Evaluate alignment folding</strong>. When this option is enabled, matches having E-value &lt; <strong>Inclusion E-value</strong> 
                                are analyzed using <a href="https://www.tbi.univie.ac.at/RNA/RNAalifold.1.html">RNAalifold</a> to derive a consensus structure for the match
                                alignment (<strong>Note:</strong> this operation can increase the computation time)
                            </li>
                        </ul>
                        <br/><br/>
                        <strong><u>Advanced options</u></strong>
                        <br/><br/>
                        <li><strong>General</strong></li>
                        <br/>
                        <ul>
                            <li><strong>Max reactivity</strong>. Maximum value to which reactivities will be capped</li>
                            <li><strong>Max alignment overlap</strong>. If two significant alignments overlap by more than this value, the least significant one (the one with the lowest alignment score) will be discarded</li>
                            <li><strong>Null HSGs</strong>. Maximum number of HSGs in the shuffled database to be extended to build the null model</li>
                        </ul>
                        <br/><br/>
                        <li><strong>Kmer lookup</strong></li>
                        <br/>
                        <ul>
                            <li><strong>Min kmers</strong>. Minimum number of kmers required to form a High Scoring Group</li>
                            <li><strong>Max kmer distance</strong>. Maximum distance between two kmers to be merged in a HSG</li>
                            <li><strong>Kmer length</strong>. Length (in nt) of the kmers</li>
                            <li><strong>Kmer offset</strong>. Sliding offset (in nt) for extracting candidate kmers from the query</li>
                            <li><strong>Min kmer complexity</strong>. Minimum complexity (measured as Gini coefficient) of candidate kmers</li>
                            <li><strong>Kmer max match every nt</strong>. A kmer is allowed to match a database entry on average every this many nt</li>
                            <li><strong>Match kmer GC content</strong>. The sequence of a query kmer and the corresponding database match must have GC% contents differing no more than <strong>Kmer max GC difference</strong></li>
                            <li><strong>Kmer max GC difference</strong>. Maximum allowed GC% difference to retain a kmer match</li>
                            <li><strong>Match kmer sequence</strong>. The sequence of a query kmer and the corresponding database match must differ no more than <strong>Kmer max sequence distance</strong></li>
                            <li>
                                <strong>Kmer max sequence distance</strong>. Maximum allowed sequence distance to retain a kmer match. When &ge; 1, this is interpreted as the absolute number of bases that are allowed to differ 
                                between the kmer and the matching region. When &lt; 1, this is interpreted as a fraction of the kmer's length
                            </li>
                        </ul>
                        <br/><br/>
                        <li><strong>Alignment</strong></li>
                        <br/>
                        <ul>
                            <li><strong>Match score</strong>. Minimum and maximum score reactivity differences below 0.5 will be mapped to</li>
                            <li><strong>Mismatch score</strong>. Minimum and maximum score reactivity differences above 0.5 will be mapped to </li>
                            <li><strong>Gap open penalty</strong>. Penalty for opening a gap</li>
                            <li><strong>Gap extension penalty</strong>. Penalty for extending a gap</li>
                            <li><strong>Max drop-off rate</strong>. An alignment is allowed to drop by maximum this fraction of the best score encountered so far, before extension is interrupted </li>
                            <li><strong>Max drop-off bases</strong>. An alignment is allowed to drop below the best score encountered so far &times; <strong>Max drop-off rate</strong> by this number of bases, before extension is interrupted</li>
                            <li><strong>Length difference tolerance</strong>. The maximum allowed tollerated length difference between the query and db sequences to look for the ideal alignment along the diagonal (measured as a fraction of the length of the shortest sequence between the db and the query)</li>
                            <li><strong>Score sequence</strong>. Sequence will be also considered during the alignment</li>
                            <li><strong>Sequence match score</strong>. Score reward for matching bases</li>
                            <li><strong>Sequence mismatch score</strong>. Score penalty for mismatching bases</li>
                        </ul>
                        <br/><br/>
                        <li><strong>Folding evaluation</strong></li>
                        <br/>
                        <ul>
                            <li><strong>Shufflings</strong>. Number of shufflings to perform for each alignment</li>
                            <li><strong>Block size</strong>. Size (in nt) of the blocks for shuffling the alignment</li>
                            <li><strong>In-block shuffle</strong>. Besides shuffling blocks, residues within each block will be shuffled as well</li>
                            <li><strong>Min base-pair support</strong>. Minimum fraction of base-pairs of the RNAalifold-inferred structure that should be supported by both query and matched database sequence to retain a match</li>
                            <li><strong>Max base-pair span</strong>. Maximum allowed base-pairing distance in the consensus structure</li>
                            <li><strong>Slope</strong>. Slope for SHAPE reactivities conversion into pseudo-free energy contributions</li>
                            <li><strong>Intercept</strong>. Intercept for SHAPE reactivities conversion into pseudo-free energy contributions</li>
                            <li><strong>No lonely pairs</strong>. Disallows lonely pairs (helices of 1 bp) in the consensus structure</li>
                            <li><strong>No closing GU</strong>. Dissalows G:U wobbles at the end of helices in the consensus structure</li>
                            <li><strong>RIBOSUM scoring</strong>. Use RIBOSUM scoring matrix (please refer to the <a href="https://www.tbi.univie.ac.at/RNA/RNAalifold.1.html">RNAalifold manual</a> for details)</li>
                        </ul>
                    </ul>
                </p>
                <br/>
                <h1 class="content-title" id="understanding-search-parameters">Understanding search parameters</h1>
                <p>
                    While it is advisable for most users to run SHAPEwarp with its default parameters, as these are the results of a careful 
                    and thorough calibration, it might be useful to adjust the analysis on a case-by-case basis.<br/><br/>
                    First of all, as SHAPE reactivities can have a variable range, depending on the adopted <a href="https://rnaframework-docs.readthedocs.io/en/latest/rf-norm/#normalization-of-raw-reactivities">normalization scheme</a>, 
                    both query and database reactivities are capped to a maximum value of <strong>Max reactivity</strong>. 
                    During the <i>kmer lookup</i> phase, the algorithm enumerates all the possible kmers of size <strong>Kmer length</strong> in the user-provided query, filtering out those:<br/><br/>
                    <strong>1.</strong> encompassing <i>NaN</i> values<br/>
                    <strong>2.</strong> having Gini coefficient &lt; <strong>Min kmer complexity</strong>, corresponding to regions of low structural complexity (expected to match most transcripts)<br/><br/>
                    Kmers are matched against the database using the <a href="https://www.cs.unm.edu/~mueen/FastestSimilaritySearch.html">MASS</a> algorithm. Given a kmer and a database reactivity profile, 
                    MASS returns an array of distances of that kmer to each position of the database profile. Matches having a distance &lt; &mu; - 3 &times; &sigma;, where &mu; and &sigma; are respectively 
                    the mean and the standard deviation of the distances, are retained. Kmer-match pairs are then filtered out on the basis of the following criteria:<br/><br/>
                    <strong>1.</strong> when <strong>Match kmer GC content</strong> is enabled and the difference between the GC% content of the kmer and that of its database match is &gt; <strong>Kmer max GC difference</strong><br/>
                    <strong>2.</strong> when <strong>Match kmer sequence</strong> is enabled and the sequence of the kmer and that of its database match differ by more than <strong>Kmer max sequence distance</strong><br/>
                    <strong>3.</strong> when the ratio between the length of the database transcript and the number of matches for a given kmer in that transcript is &lt; <strong>Kmer max match every nt</strong><br/>
                    The retained kmers are then grouped into high-scoring groups (HSGs). HSGs are made of kmers residing on the same diagonal (so, corresponding to regions of the query and database that can be theoretically aligned 
                    without opening gaps). Each HSG is made of &ge; <strong>Min kmers</strong>, residing within a maximum distance of <strong>Max kmer distance</strong> from each other.<br/><br/>
                    Each HSG constitutes the <i>seed</i> from which an alignment extension will be attempted, both upstream and downstream. The <i>seed extension</i> uses a dynamic programming algorithm that incorporates features 
                    of both dynamic time warping (DTW) and of Gotoh's Smith-Waterman implementation with affine gap penalties. In brief, if the reactivity difference between two bases is &lt; 0.5, the corresponding alignment score 
                    is calculated by linearly mapping the reactivity difference to the <strong>Match score</strong> range. Similarly, if the reactivity difference is &ge; 0.5, the corresponding alignment score is calculated by 
                    linearly mapping the reactivity difference to the <strong>Mismatch score</strong> range. Affine gap penalties are controlled via the <strong>Gap open penalty</strong>, the gap open penalty, and <strong>Gap extension penalty</strong>, 
                    the gap extension penalty. During the matrix-filling stage of the algorithm, the alignment score is allowed to drop by not more than <strong>Max drop-off rate</strong> &times; the best alignment score encountered so far, 
                    for no more than <strong>Max drop-off bases</strong>. While the algorithm is sequence-agnostic by default, sequence can be optionally taken into account by enabling the <strong>Score sequence</strong> option. 
                    When enabled, sequence matches are rewarded by <strong>Sequence match score</strong> and mismatches are penalized by <strong>Sequence mismatch score</strong>. To further speed-up the alignment phase, 
                    the alignment matrix is only filled in a band of size <i>b</i> around the diagonal, where <i>b</i> is defined as min(10, <strong>Length difference tolerance</strong> &times; min(<i>q</i>, <i>d</i>)), where <i>q</i> and 
                    <i>d</i> are respectively the length of the query and of the database entry. Therefore, a minimum band size of 10 nt is always granted. The final alignment score <i>S</i> is calculated as the sum of the scores of the HSG 
                    (the <i>seed</i>), plus the two upstream and downstream extensions.<br/><br/>
                    After having extended any possible HSG in the database, the same procedure is repeated by searching the query in a database of shuffled reactivity profiles. A maximum of <strong>Null HSGs</strong> HSGs identified in the shuffled 
                    database are extended, and the resulting alignment scores are used to build the null model, hence allowing to calculate a <i>p-value</i> for each alignment from the real database. The <i>p-value></i> is the probability of 
                    identifying by chance a match having a score &ge; <i>S</i>. The <i>E-value</i> corresponds to the number of expected database matches having a score &ge; <i>S</i> and it is thus calculated as <i>p-value</i> &times; the number of extended HSGs. 
                    It might happen, especially with long queries, that multiple HSGs will be identified for the same query-database entry pair, hence resulting in multiple overlapping alignments. To prevent duplicate matches from being reported, 
                    if multiple alignments overlap by more than <strong>Max alignment overlap</strong>, only the one with the lowest E-value is reported.<br/><br/>
                    Optionally, if the <strong>Evaluate alignment folding</strong> option is enabled, alignments passing the <strong>Inclusion E-value</strong> cutoff can be evaluated for the presence of a consensus secondary structure, via <a href="https://www.tbi.univie.ac.at/RNA/RNAalifold.1.html">RNAalifold</a>. 
                    RNAalifold uses both the sequence alignment and the corresponding SHAPE reactivity information. SHAPE reactivities are converted into pseudo free energy contributions by using the approach from <a href="https://pubmed.ncbi.nlm.nih.gov/19109441/">Deigan *et al.*, 2009</a>, where:<br/><br/>
                    &#916;G<sub>SHAPE</sub>(<i>i</i>) = <strong>Slope</strong> &middot; ln[Reactivity(<i>i</i>) + 1] + <strong>Intercept</strong>
                    <br/><br/>
                    Pre-defined <strong>Slope</strong>/<strong>Intercept</strong> value pairs are available for standard SHAPE reagents.<br/>
                    If &lt; than <strong>Min base-pair support</strong> base-pairs from the predicted consensus structure are supported for either of the two aligned RNAs, the match is discarded. Furthermore, the probability of predicting a structure with a free 
                    energy &le; to that of the consensus structure inferred from the alignment is evaluated by randomly shuffling <strong>Shufflings</strong> times the columns of the original alignment in <strong>Block size</strong>-long blocks, followed by RNAalifold analysis. 
                    A z-score is calculated and the corresponding <i>p-value</i> determined. If the <i>p-value</i> is &lt; 0.05, the alignment and the inferred structure are reported.<br/>
                </p>
                <br/>
                <h1 class="content-title">SHAPEwarp results</h1>
                <p>
                    SHAPEwarp will return a table containing the list of all the significant matches (so, the matches within the user-defined 
                    <strong>inclusion E-value</strong> threshold). These results will be marked by a "<strong>!</strong>" under the "<strong>Significance</strong>" 
                    column. If a <strong>report E-value</strong> higher than the inclusion E-value was specified, matches above the inclusion E-value threshold, 
                    but below the report E-value threshold, will also be reported, marked by a "<strong>?</strong>" and colored in red.
                </p>
                <p>
                    A <a href="scripts/results.cgi?jobId=samplequery-000000">sample SHAPEwarp output</a> is depicted in picture below:
                    <br/><br/>
                    <img src="images/sample_results.png" alt="Sample results"/>
                    <br/>
                    By clicking on each result, a detailed overview of the specific match will be shown:
                    <br/><br/>
                    <img src="images/sample_result_expanded.png" alt="Sample result expanded"/>
                    <br/>
                    Four (or five, depending on whether the "<strong>Evaluate alignment folding</strong>" option was enabled) tabs will be available:
                </p>
                <p>
                    <ul>
                        <li><strong>Details.</strong> A detailed summary of the specific match</li>
                        <li><strong>Alignment.</strong> The sequence alignment between the query and the database match</li>
                        <li><strong>Reactivity plot.</strong> Plot depicting the aligned SHAPE reactivity profiles between the query and the database match</li>
                        <li><strong>Structure.</strong> Plot of the consensus structure as derived by <a href="https://pubmed.ncbi.nlm.nih.gov/17993696/">RNAalifold</a>, colored either by the query or database match SHAPE reactivities (<i>Note: structure plot generation requires the "<strong>Evaluate alignment folding</strong>" option to be enabed</i>) </li>
                        <li><strong>Downloads.</strong> Links to download the aligned sequences and reactivity profiles (as well as the structure plots, when present) for the query and the database match</li>
                    </ul>
                </p>
                <br/>
                <h1 class="content-title">Available databases</h1>
                <p>
                    SHAPEwarp provides a number of high-quality SHAPE reactivity databases. Additional databases will be added over time. If you want to request the addition of a specific database, please <a href="mailto:dincarnato[at]incarnatolab.com">contact us</a>.
                </p>
                <p>
                    <ul>
                        <li><strong>Mammals</strong></li>
                        <ul>
                            <li>
                                <i>Sun et al.</i>, 2021. <a href="https://pubmed.ncbi.nlm.nih.gov/33623109/"><strong>Predicting dynamic cellular protein-RNA interactions by deep learning using in vivo RNA structures.</strong></a> (Cell Res.)<br/>
                                <strong>RNA class:</strong> PolyA+<br/>
                                <strong>Reagent:</strong> NAI-N3<br/>
                                <strong>Library strategy:</strong> icSHAPE<br/>
                                <strong>Condition:</strong> <i>in vivo</i><br/>
                                <strong>Dataset(s):</strong> Homo sapiens (HEK293, HeLa, HepG2, H9 hESC, K562); Mus musculus (mES)
                            </li>
                            <br/>
                            <li>
                                <i>Monroy-Eklund et al.</i>, 2023. <a href="https://pubmed.ncbi.nlm.nih.gov/36792358/"><strong>Structural analysis of MALAT1 long noncoding RNA in cells and in evolution.</strong></a> (RNA)<br/>
                                <strong>RNA class:</strong> MALAT1 lncRNA<br/>
                                <strong>Reagent:</strong> 5NIA<br/>
                                <strong>Library strategy:</strong> SHAPE-MaP<br/>
                                <strong>Condition:</strong> <i>in vivo</i> &amp; <i>ex vivo</i><br/>
                                <strong>Dataset(s):</strong> Homo sapiens (A549, HEK293); Chlorocebus sabeus (VERO)
                            </li>
                        </ul>
                        <br/>
                        <li><strong>Bacteria</strong></li>
                        <ul>
                            <li>
                                <i>Mustoe et al.</i>, 2018. <a href="https://pubmed.ncbi.nlm.nih.gov/29551268/"><strong>Pervasive Regulatory Functions of mRNA Structure Revealed by High-Resolution SHAPE Probing.</strong></a> (Cell)<br/>
                                <strong>RNA class:</strong> mRNA<br/>
                                <strong>Reagent:</strong> 1M7<br/>
                                <strong>Library strategy:</strong> SHAPE-MaP<br/>
                                <strong>Condition:</strong> <i>in vivo</i><br/>
                                <strong>Dataset(s):</strong> Escherichia coli
                            </li>
                        </ul>
                        <br/>
                        <li><strong>Viruses</strong></li>
                        <ul>
                            <li>
                                <i>Li et al.</i>, 2018. <a href="https://pubmed.ncbi.nlm.nih.gov/30472207/"><strong>Integrative Analysis of Zika Virus Genome RNA Structure Reveals Critical Determinants of Viral Infectivity.</strong></a> (Cell Host Microbe.)<br/>
                                <strong>RNA class:</strong> gRNA<br/>
                                <strong>Reagent:</strong> NAI-N3<br/>
                                <strong>Library strategy:</strong> icSHAPE<br/>
                                <strong>Condition:</strong> <i>in vivo</i><br/>
                                <strong>Dataset(s):</strong> Zika virus (Asian and African strains)
                            </li>
                            <br/>
                            <li>
                                <i>Manfredonia et al.</i>, 2020. <a href="https://pubmed.ncbi.nlm.nih.gov/33166999/"><strong>Genome-wide mapping of SARS-CoV-2 RNA structures identifies therapeutically-relevant elements.</strong></a> (Nucleic Acids Res.)<br/>
                                <strong>RNA class:</strong> gRNA<br/>
                                <strong>Reagent:</strong> NAI<br/>
                                <strong>Library strategy:</strong> SHAPE-MaP<br/>
                                <strong>Condition:</strong> <i>in vivo</i> &amp; <i>in vitro</i><br/>
                                <strong>Dataset(s):</strong> SARS-CoV-2
                            </li>
                            <br/>
                            <li>
                                <i>Morandi et al.</i>, 2022. <a href="https://pubmed.ncbi.nlm.nih.gov/35361788/"><strong>SHAPE-guided RNA structure homology search and motif discovery.</strong></a> (Nat. Commun.)<br/>
                                <strong>RNA class:</strong> gRNA<br/>
                                <strong>Reagent:</strong> 2A3<br/>
                                <strong>Library strategy:</strong> SHAPE-MaP<br/>
                                <strong>Condition:</strong> <i>in vitro</i><br/>
                                <strong>Dataset(s):</strong> SARS-CoV; SARS-CoV-2
                            </li>
                            <br/>
                            <li>
                                <i>Yang et al.</i>, 2024. <a href="https://pubmed.ncbi.nlm.nih.gov/38416642/"><strong>Mapping of the influenza A virus genome RNA structure and interactions reveals essential elements of viral replication.</strong></a> (Cell Rep.)<br/>
                                <strong>RNA class:</strong> gRNA<br/>
                                <strong>Reagent:</strong> NAI-N3<br/>
                                <strong>Library strategy:</strong> icSHAPE<br/>
                                <strong>Condition:</strong> <i>in vivo</i> &amp; <i>in virio</i><br/>
                                <strong>Dataset(s):</strong> Influenza A virus
                            </li>
                        </ul>
                        <br/>
                        <li><strong>Other</strong></li>
                        <ul>
                            <li>
                                <i>Marinus et al.</i>, 2021. <a href="https://pubmed.ncbi.nlm.nih.gov/33398343/"><strong>A novel SHAPE reagent enables the analysis of RNA structure in living cells with unprecedented accuracy.</strong></a> (Nucleic Acids Res.)<br/>
                                <strong>RNA class:</strong> rRNA<br/>
                                <strong>Reagent:</strong> 2A3<br/>
                                <strong>Library strategy:</strong> SHAPE-MaP<br/>
                                <strong>Condition:</strong> <i>in vivo</i><br/>
                                <strong>Dataset(s):</strong> Escherichia coli; Bacillus subtilis; Saccharomyces cerevisiae; Homo sapiens (HEK293)
                            </li>
                        </ul>
                    </ul>
                </p>
                <br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>
            </div>
        </section>
        <footer>
            <div class="container">
                <a href="https://www.incarnatolab.com/">
                    <h1 class="page-title" style="font-size: 1.2rem;">INCA<span class="page-title2" style="font-size: 1.8rem;">RNA</span>TO lab</h1>
                </a>
            </div>
        </footer>
    </body>
</html>